### PR TITLE
Bump requirements

### PR DESCRIPTION
**Pull request by xbrianh**

### CODE REVIEW

File: chalice/app.py [hunk 1]
```diff
@@ -202,7 +202,7 @@ def maybe_fake_504() -> bool:
             with flask_app.test_request_context(
                     path=path,
                     base_url="https://{}".format(app.current_request.headers["host"]),
-                    query_string=app.current_request.query_params,
+                    query_string=list((app.current_request.query_params or dict()).items()),
```
Comment (amarjandu) [inline chalice/app.py]: why the change here?

Comment (xbrianh) [reply chalice/app.py]: Needed to work with the new version of chalice.

**Comment by xbrianh** [comment]
@amarjandu The requirement bumps are the result of `make refresh_all_requirements`. The .in files were not changed.
